src/Zadatak5/NejednakostranicniTrougao.java

```diff
@@ -0,0 +1,61 @@
+package Zadatak5;
+
+/**
+ * Zadatak 5
+ *
+ * @author Vladimir.Krekic
+ */
+
+class NejednakostranicniTrougao extends Trougao{
+
+    private double stranica2;
+    private double stranica3;
+    private static final String NAZIV = "Nejednakostranicni trougao";
+
+    public NejednakostranicniTrougao() {
+    }
+
+    public NejednakostranicniTrougao(double stranica, double stranica2, double stranica3) {
+        super(stranica);
+        this.stranica2 = stranica2;
+        this.stranica3 = stranica3;
+    }
+
+    public double getStranica2() {
+        return stranica2;
+    }
+
+    public void setStranica2(double stranica2) {
+        this.stranica2 = stranica2;
+    }
+
+    public double getStranica3() {
+        return stranica3;
+    }
+
+    public void setStranica3(double stranica3) {
+        this.stranica3 = stranica3;
+    }
+
+    public double visina(){
```

Comment:
Po kojoj formuli tacno racunas visinu?

Mislim da bi ti bilo lakse da is ozracunao povrsinu po heronovm obrascu.